### PR TITLE
🧪 Add tests for benchmarking script

### PR DESCRIPTION
**Pull request by ashwinvaidya17**
# Description

- Adds test to benchmarking script. Increases the test time by ~3mins. Should I move this to nightly? 

## Changes

- Add test for benchmarking script

## Checklist

- [x] My code follows the [pre-commit style and check guidelines](https://openvinotoolkit.github.io/anomalib/guides/using_pre_commit.html#pre-commit-hooks) of this project.
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing tests pass locally with my changes

### CODE REVIEW

**Review by samet-akcay — COMMENTED** [review]
I would agree that benchmarking could go to the nightly tests.

File: tests/pre_merge/tools/benchmarking/test_benchmarking.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+
+from pathlib import Path
+
+from omegaconf import OmegaConf
+
+from tests.helpers.dataset import get_dataset_path
+
+
+def check_tf_logs(model: str):
```
Comment (samet-akcay) [inline tests/pre_merge/tools/benchmarking/test_benchmarking.py:36]: should this be `check_tb_logs`?

Comment (ashwinvaidya17) [reply tests/pre_merge/tools/benchmarking/test_benchmarking.py:36]: Yeap! My bad

**Comment by ashwinvaidya17** [comment]
> I would agree that benchmarking could go to the nightly tests.

I'll move it to nightly once CI passes

**Review by samet-akcay — APPROVED** [review]

**Pull request merged**